packages/styled-system/README.md

```diff
@@ -11,3 +12,90 @@ npm i @chakra-ui/styled-system
 
 yarn add @chakra-ui/styled-system
 ```
+
+### CSS Logical Properties
+
+Chakra UI provides pretty good support for bi-directional (`bidi`) CSS
+properties. For our shorthand props, we provide a `*Bidi` suffix to manage the
+rtl/ltr switching.
+
+```jsx
+import { css } from "@chakra-ui/styled-system"
+
+const theme = {
+  direction: "rtl", // ltr | rtl
+}
+
+const styles = css({
+  mt: "40px",
+  // bi-directional `margin-right`
+  // `margin-right` in ltr, `margin-left` in rtl
+  mrBidi: "30px",
```

Comment:
_"I can't imagine building out a system that needs to support both ltr and rtl and having to explicitly set values for both versions in every case."_

**Re:** You won't have to define values for both rtl and ltr values. The goal of this is to introduce css logical props for RTL users to use that will handle both cases automatically.

We could add this implicitly but I'm careful not to touch existing, stable functionality. Emotion's `CacheProvider` already has a way to add rtl plugins to change the layout to RTL (writing a guide for this).

Most RTL plugins or systems have a way to opt for the automatic layout mirroring by adding a `noflip` comment to the CSS values but I don't want to delve into that as well. Hence the reason to introduce new props that mirror existing props but with a suffix or prefix.

I can also get rid of the shorthand "*Bidi" props all together to avoid confusion.

Reply:
@chakra-ui/react-team What are your thoughts?